Channels/UserInterface/04a_OfferImport.md

```diff
@@ -0,0 +1,97 @@
+
+      - ![Save](../../Assets/Icons/Save.png "[Save]") save    
+        Click this entry to save the current view settings in the selected view. This menu entry is only displayed if a view has been selected.
+
+        > [Info] When the settings of a view have been changed, an asterisk is displayed next to the view name. The asterisk is hidden as soon as the changes have been saved.
+
+      - ![Delete](../../Assets/Icons/Trash01.png "[Delete]") delete    
+        Click this entry to delete the selected view. A confirmation window is displayed to confirm the deletion. This menu entry is only displayed if a view has been selected.
+
+- *Select Connection*    
+  Click the drop-down list to select a connection. All supported connections are displayed in the list. You must select a connection to display any offer imports in the list.
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+  Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.
+
+- ![Filter](../../Assets/Icons/Filter.png "[Filter]") Filter (x)   
+Click this button to display the filter bar and customize the active filters. The *x* indicates the number of filters that are currently active. This button is only displayed if a connection has been selected in the *Select connection* drop-down list.
+
+- [x]     
+  Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all offer imports in the list are selected.
+
+- [IMPORT PRODUCT(S) TO CHANNELS]   
+  Click this button to import the selected offer(s) to the *Omni-Channel* module. This button is only displayed if a connection has been selected in the *Select connection* drop-down list and if the checkbox of at least one offer is selected.
+
+- [DETAILS]   
+  Click this button to display the *Detail of log message "Log message title"* view and check further details of the error. This button is only displayed if a connection has been selected in the *Select connection* drop-down list and if the checkbox of a single offer with an error is selected.
+
+
+The list displays all offer imports. Depending on the settings, the displayed columns may vary. All fields are read-only. If no offer imports are available, the *Please select a connection to import products from* notice is displayed. The available columns differ depending on the selected connection in the *Select Connection* drop-down list. Therefore, only those columns that are available for all connections are described in the following.
+
+- *Error message*   
+  Error log message if an error has occurred. Select the checkbox of the corresponding row and click the [DETAILS] button in the editing toolbar to display further details to the error. If no error has occurred, the field is empty.
```

Comment:
If no error has occurred, the field is empty. --> Welches Feld? Ich kann das nicht überprüfen, denn es gibt keine Fehler bei mir, aber wieso ist das möglich? Wenn es keinen Fehler gibt, sollte es auch keine Error message geben, oder?